tern/report/report.py

```diff
@@ -154,5 +156,9 @@ def report_out(args, *images):
         logger.error("%s not a recognized plugin.", args.report_format)
     elif args.output_file:
         write_report(report, args)
+        if args.report_format == "json":
+            create_html(json.loads(report),images,args,filename=args.output_file)
```

Comment:
When the user does not specify an output file on the command line (i.e. `tern report -i ubuntu:latest -f json`), the file `report.html.html` is created.